test/form_builder/error_test.rb

```diff
@@ -272,4 +272,14 @@ def with_full_error_for(object, *args)
       assert_no_select 'span.error'
     end
   end
+
+  test 'input wrapper has valid class for present attribute without errors' do
```

Comment:
We could add wrapper test as well, like this one:

https://github.com/plataformatec/simple_form/blob/31720f00f9ce482e8c895fba6707cbdea08710cf/test/form_builder/wrapper_test.rb#L36

Wdty?

Reply:
This test is currently only testing the wrapper div. I'll move it to the `wrapper_test.rb`

```html
<form class="simple_form edit_user" id="edit_user_1" action="/users" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="&#x2713;" /><input type="hidden" name="_method" value="patch" /><div class="input string required user_name field_without_errors"><label class="string required" for="user_name"><abbr title="required">*</abbr> Super User Name!</label><input class="string required" required="required" aria-required="true" maxlength="100" size="100" type="text" value="New in SimpleForm!" name="user[name]" id="user_name" /></div></form>
```